superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -10,8 +10,9 @@
 from superbench.benchmarks.micro_benchmarks.cublas_function import CublasBenchmark
 from superbench.benchmarks.micro_benchmarks.cudnn_function import CudnnBenchmark
 from superbench.benchmarks.micro_benchmarks.gemm_flops_performance import GemmFlopsCuda
+from superbench.benchmarks.micro_benchmarks.cuda_memory_bw_performance import CudaMemBwBenchmark
```

Comment:
'cuda_memory_bw_performance' => 'memory_bw_performance'?
we can put rocm and cuda implementation into the same file.

Reply:
Yes, it's my key question. Whether we would like to merge AMD in this? Or we can have RocmMemBwBenchmark. Would it be easy to maintain the code?

Reply:
I have thought about it before:

Option1:
```
superbench/benchmarks/micro_benchmarks/
   --CUDA
       --memory_bw_performance.py
   --ROCm
       --memory_bw_performance.py
```

For those benchmarks that have no difference between CUDA and ROCm, like 1. pure python benchmarks 2. benchmarks we can control the input/output (kernel launch), where should we put them? Just into superbench/benchmarks/micro_benchmarks/?

Option2:
```
superbench/benchmarks/micro_benchmarks/
    --memory_bw_performance.py (has both cuda and rocm)
```

For the reason that I commented to use Option2:
1. keep the same benchmark registry in the same file:
BenchmarkRegistry.register_benchmark('mem-bw', MemBwCuda, platform=Platform.CUDA)
BenchmarkRegistry.register_benchmark('mem-bw', MemBwRocm, platform=Platform.ROCm)
2. No need to change the directory structure in this PR.

Option3:
```
superbench/benchmarks/micro_benchmarks/
    --cuda_memory_bw_performance.py
    --rocm_memory_bw_performance.py
```

I thought put cuda_xxx, rocblas_xxx, cudnn_xxx, cublas_xxx, rocm_xxx, rocblas_xxx into one folder is a bit messy.

We can consider using option3 currently. In the long term, we may consider Option1.